spec/models/file_download_stat_spec.rb

```diff
@@ -78,11 +78,13 @@
         described_class.statistics(file, Time.zone.today - 4.days)
       end
 
-      it "includes cached ga data" do
+      # TODO: Fix this test when analytics work is picked up again
+      skip it "includes cached ga data" do
```

Comment:
How do we make sure these don't get forgotten?

Reply:
I just made a ticket for it so it will be on the board when analytics gets picked back up

https://github.com/scientist-softserv/palni-palci/issues/559